readthedocs/settings/base.py

```diff
@@ -1085,3 +1085,8 @@ def SOCIALACCOUNT_PROVIDERS(self):
     }
 
     S3_PROVIDER = "AWS"
+    AWS_STS_ASSUME_ROLE_ARN = "arn:aws:iam::1234:role/SomeRole"
```

Comment:
This definitely needs a comment and explainer. The docstring in readthedocs/storage/security_token_service.py is a good place to link to, or add that to the dev docs and link there.

Reply:
```suggestion
    # Used by readthedocs.aws.security_token_service.
    # It's constructed as "arn:aws:iam:::{bucket name}:{path}"
    AWS_STS_ASSUME_ROLE_ARN = "arn:aws:iam:::1234:role/SomeRole"
```

Also note that we were using `::` (two) here but in the code we are using 3.

Reply:
This is the role's ARN, that's two semicolons, bucket's ARN's have three.